### PR TITLE
PE Controls #247

### PR DESCRIPTION
**Pull request by markXLIX**
If PE controls exist and they are inherited from a leveraged authorization, throw a warning.

### CODE REVIEW

**Comment by markXLIX** [comment]
This won't show in the SSP Validator because an associated PE control with leveraged authorization does not exist.  It was tested using an AC control which does have a leveraged authorization.  The xspec test passes.

**Review by GaryGapinski — COMMENTED** [review]
We can discuss this at our convenience. I do not think that **non-Schematron** constructs (i.e., XSLTones ) should be used unless there is a good reason.

**Comment by markXLIX** [comment]
Got it.  For ease of maintenance for whomever needs to deal with this code later I will avoid xslt structures where at all possible.  Makes me slightly sad for the loss of xsl:key but we work within the framework.  ;-)

**Comment by markXLIX** [comment]
@GaryGapinski Can you take another look?

File: src/validations/rules/ssp.sch [hunk 1]
```diff
@@ -523,7 +523,15 @@
                 test="true()">A FedRAMP SSP must implement a statement for each of the following lettered response points for required controls: <sch:value-of
                     select="$implemented/@statement-id" />.</sch:report>
         </sch:rule>
-        
+        <sch:rule
```
Comment (GaryGapinski) [inline src/validations/rules/ssp.sch:526]: You encountered a bothersome condition which I repeatedly had last year when multiple PRs were in flight.

This commit is the best way to fix such (when such a commit is sufficient).

**Pull request merged**